src/Raven.Server/Documents/PeriodicBackup/Retention/AzureRetentionPolicyRunner.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Server.Documents.PeriodicBackup.Azure;
+
+namespace Raven.Server.Documents.PeriodicBackup.Retention
+{
+    public class AzureRetentionPolicyRunner : RetentionPolicyRunnerBase
+    {
+        private readonly RavenAzureClient _client;
+
+        protected override string Name => "Azure";
+
+        public AzureRetentionPolicyRunner(RetentionPolicy retentionPolicy, string databaseName, Action<string> onProgress, RavenAzureClient client)
+            : base(retentionPolicy, databaseName, onProgress)
+        {
+            _client = client;
+        }
+
+        protected override Task<List<string>> GetFolders()
+        {
+            throw new NotSupportedException();
```

Comment:
Create a separate issue to support that.

Reply:
Done